pkg/controller/servicebindingrequest/annotations_test.go

```diff
@@ -3,7 +3,7 @@ package servicebindingrequest
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
I think we can use both assert and require. Use "require" for absolute required steps that are required for further assertions. See the below comment for an example.

Reply:
I agree with this principle, let's continue on the other related comment.